### ISSUE TITLE
MathCAT userguide discrepancies with NVDA menus

### ISSUE THREAD

**Issue by qoala** [NSoiffer/MathCATForPython#114]
NVDA version: 2026.2.0.57664
I've built the addon from source earlier this week and installed it via the "install from external source" button in NVDA's addon store.

---

Looking at the instructions in https://daisy.github.io/MathCAT/users.html#adapt-mathcat-to-your-needs
I've noticed two discrepancies.

1.
    > You can find the settings by pressing the NVDA-key+N, then Settings, then MathCAT settings.
 
   In the left navigation pane, the relevant entry is labelled "Math", not "MathCAT".

2. The user guide describes an option for "Speech" > "Disability", \
    but within NVDA, I see "Speech" > "Impairment", with the same options available as documented.

---

Are these issues that need to be updated in the documentation? Alternatively, did I not install the addon successfully (considering that a version of MathCAT is now included by default with NVDA)?

@rileeki

**Comment by NSoiffer** [comment]
If the addon is installed, then at the bottom of the NVDA preferences menu, there should be a "MathCAT Settings..." entry. In all cases, the builtin version has a "Math" tab under NVDA's "Preferences" dialog. Potentially the addon could remove it (maybe). Another thing the addon _maybe_ should do is sync the settings between the two places. NVDA renamed a few settings because... I don't know.
